studies/resources/full_experiment.py

```diff
@@ -124,22 +124,37 @@ def dehydrate_finding_tags(self, experiment: Experiment):
         )
 
     def resolve_finding_tag_description(self, finding: FindingTag):
-        return INNER_SEPERATOR.join(
-            [
-                f"type: {finding.type.name}",
-                f"family: {finding.family.name}",
-                f"is_NCC: {finding.is_NCC}",
-                f"atlas_tag: {finding.AAL_atlas_tag}",
+        base_finding_resources = [
+            f"type: {finding.type.name}",
+            f"family: {finding.family.name}",
+            f"is_NCC: {finding.is_NCC}",
+            f"technique: {finding.technique}",
+            f"notes: {finding.notes}",
+        ]
+        if finding.family.name.lower() == "spatial" and finding.technique.name.lower == "fmri":
```

Comment:
needs to be `finding.technique.name.lower()`

Reply:
true, would fix